runtime/README.md

```diff
@@ -281,6 +281,35 @@ Command Code: `0x5349_4756` ("SIGV")
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips\_status | u32      | Indicates if the command is FIPS approved or an error.
 
+### LMS\_SIGNATURE\_VERIFY
```

Comment:
For the parameters, we should be clear what each thing refers too (maybe referencing https://datatracker.ietf.org/doc/html/rfc8554).

For example it's not really clear which part of the public key the key id is.

Reply:
I can  include that "I" and "T[1]" notation in the description if that would make it clearer. I was building on the name/descriptions from the LMS fields in the [image format](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#firmware-image-format-and-verification).

Reply:
In that case, it would also be fine if you just want to reference sections from the image format doc (or just replicate those format descriptions here. Maybe that's better to avoid the cross-reference).

Reply:
Updated descriptions look good.